test/pkg/environment/common/expectations.go

```diff
@@ -267,22 +267,33 @@ func (env *Environment) ExpectExists(obj client.Object) {
 	}).WithTimeout(time.Second * 5).Should(Succeed())
 }
 
+func (env *Environment) EventuallyExpectBound(pods ...*v1.Pod) {
+	GinkgoHelper()
+	Eventually(func(g Gomega) {
+		for _, pod := range pods {
+			g.Expect(env.Client.Get(env, client.ObjectKeyFromObject(pod), pod)).To(Succeed())
+			g.Expect(pod.Spec.NodeName).ToNot(BeEmpty())
```

Comment:
I'm assuming this is the same as == ""?

Reply:
Well, it's the same as `!= ""` but yeah that's correct.